.vib/magento/goss/vars.yaml

```diff
@@ -0,0 +1,22 @@
+binaries:
+  - apachectl
+  - php
+  - magento
+  - mysql
+  - php
+  - render-template
+  - wait-for-port
+  - composer
+directories:
+  - mode: "0775"
+    paths:
+      - /opt/bitnami/magento
+files:
+  - paths:
+      # Validate that a supported web server is configured
+      - /opt/bitnami/scripts/apache-env.sh
+      - /opt/bitnami/scripts/libapache.sh
```

Comment:
I'm not sure we need to check these files, as they would always be present as part of the apache nami module. We want to add a check for the postunpack logic that goes by:
```
ensure_web_server_app_configuration_exists "magento" --type php \
    --apache-move-htaccess no
```
In that case, we may want to check there is indeed no htaccess file created by apache at `/opt/bitnami/apache/conf/vhosts/htaccess/magento-htaccess.conf`